pkg/cryptoEAS/index.go

```
@@ -0,0 +1,37 @@
+	ciphertext := aesgcm.Seal(nil, nonce, []byte(plaintext), nil)
+	return ciphertext, nonce
+}
+
+func decryptAES(ciphertext, key, nonce []byte) string {
+	block, _ := aes.NewCipher(key)
+	aesgcm, _ := cipher.NewGCM(block)
+	plaintext, _ := aesgcm.Open(nil, nonce, ciphertext, nil)
+	return string(plaintext)
+}
+
+func main() {
```

golangci-lint (1.64.8)

func `main` is unused

(unused)